### ISSUE TITLE
Add clubhouse link

### ISSUE THREAD

**Issue by MandyMeindersma** [devedmonton/devedmonton.com#101]
Or maybe there is an integration to see the board?

Example text:
Check out our clubhouse board: https://app.clubhouse.io/dev-edmonton-society/stories/space/148/everything
Ask Mandy, Lauren, Mark or Ian for access.

**Comment by dboydgit** [comment]
I tried to visit the link and it seems to be private? So we still want to add it to the website if it requires an account and permissions to view?

**Comment by MandyMeindersma** [comment]
Yeah when we discussed it in one of our meetings we wanted the inner workings available to the public. We have some "watcher" credentials for that board that are available by DMing myself or Ian (or maybe the board is a better idea). Hence wanting to include the full text thing above

We wanted to at least have the link up in case people wanted to see.

**Comment by MandyMeindersma** [comment]
We don't really use this anymore so closing
